### ISSUE TITLE
Progress bar UI

### ISSUE THREAD

**Issue by schriftgestalt** [elementary/installer#358]
I don’t know if this is the right repo, so please point me to the right one.

I just installed Elementary OS 5.0 Juno in a VM and was presented with this dialog. 

<img width="890" alt="screenshot" src="https://user-images.githubusercontent.com/174660/47135426-768f9b00-d2b0-11e8-9682-f68d94d0b517.png">

Nether the Layout (to much space on top) and the colors (all kind of background colors) are correct.

**Comment by schriftgestalt** [comment]
And the new one has fixed that I suppose? Thanks.

**Comment by tintou** [comment]
Yes, the new one has a completely different codebase made by the elementary team and System76
